### ISSUE TITLE
Implementation question

### ISSUE THREAD

**Issue by zhangdan8962** [facebookresearch/barlowtwins#31]
I am wondering what is the purpose for the following code block?

https://github.com/facebookresearch/barlowtwins/blob/a655214c76c97d0150277b85d16e69328ea52fd9/main.py#L90-L96

**Comment by jzbontar** [comment]
Great question. This code block partitions the parameters of the model into a) biases and batch-norm params and b) all other parameters. These two parameter groups are treated differently by the optimizer. The bias and batch-norm parameters have a [different learning rate](https://github.com/facebookresearch/barlowtwins/blob/a655214c76c97d0150277b85d16e69328ea52fd9/main.py#L167-L168) and are [excluded from weight decay and LARS adaptation](https://github.com/facebookresearch/barlowtwins/blob/a655214c76c97d0150277b85d16e69328ea52fd9/main.py#L245-L255).
